bash/create_db_sample.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
Can you add `set -e` so that errors will cause the script to exit? Probably a good habit  for the apprentices to get into